contracts/cw-ics721-bridge/src/msg.rs

```diff
@@ -0,0 +1,112 @@
+use cosmwasm_std::IbcTimeout;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+pub struct InstantiateMsg {
+    /// Code ID of cw721 contract. A new cw721 will be instantiated
+    /// for each new IBCd NFT classID.
```

Comment:
Does this have to be a special type of cw721 contract?

Reply:
Great catch. Yep! Updated the field name and improved the docs.